geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/rebalance/PartitionedRegionLoadModelJUnitTest.java

```diff
@@ -1417,9 +1467,16 @@ public void testTwoZoneRedundancyFourFullCopiesEnforceUniqueZones() throws Unkno
     assertThat(bucketOperator.removes).isEqualTo(expectedRemoves);
   }
 
-
+  /**
+   * Test that when you have five zones with one server each, with enforce unique zones, a
+   * redundancy
+   * level of 1, there is an overredundancy, and two servers have no data, that the overredundancy
+   * gets removed and buckets get moved to rebalance the data load
+   *
+   */
   @Test
-  public void testFiveZoneRedundancyFourFullCopiesEnforceUniqueZones() throws UnknownHostException {
+  public void fiveZonesWithRedundancyAndFourFullCopiesAndEnforceUniqueZones()
```

Comment:
This test name seems to be incorrect as there are three copies of each bucket, not four.

Reply:
Yup. Good catch.